app/statistics/server/lib/getServicesStatistics.ts

```diff
@@ -0,0 +1,120 @@
+import type { SettingValue } from '../../../../definition/ISetting';
+import { settings } from '../../../settings/server';
+import { Users } from '../../../models/server';
+
+function getCustomOAuthServices(): Record<string, Record<string, any>> {
+	const data: Record<string, Record<string, any>> = {};
+
+	settings.get(/Accounts_OAuth_Custom-[^-]+$/mi, function(key: string, value: SettingValue) {
+		const name = key.replace('Accounts_OAuth_Custom-', '');
+		data[name] = {
+			enabled: value,
+			mergeRoles: settings.get(`Accounts_OAuth_Custom-${ name }-merge_roles`),
+			users: Users.countActiveUsersByService(name),
+		};
+	});
+
+	return data;
+}
+
+export function getServicesStatistics(): Record<string, any> {
```

Comment:
provide something more specific than `any`, if you really dont know, use `unknown` but try to provide some kind of structure